docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -229,6 +229,78 @@ import Button from '@mui/material/Button';
 </Button>;
 ```
 
+### API
+
+`theme.applyStyles(mode, styles) => CSSObject`
+
+Apply styles for a specific mode.
+
+#### Arguments
+
+- `mode` (`'light' | 'dark'`) - The mode for which the styles should be applied.
+- `styles` (`CSSObject`) - An object which contains the styles to be applied for the specified mode.
+
+#### Returns
+
+A `CSSObject` which key is the mode selector and value is the styles object: `{ [modeSelector]: styles }`.
```

Comment:
This is true if the CSS theme variables feature is enable. However, the default behavior is still using `theme.palette.mode` to override the styles.

```js
{
  color: 'red',
  …theme.applyStyles('dark', {
    color: 'blue',
  }),
}

// result when theme.palette.mode === 'dark' is
{
  color: 'blue',
}
```